docs/troubleshooting/troubleshoot_SC4S_server.md

```diff
@@ -102,6 +102,20 @@ just _one_ bad index will "taint" the entire batch (in this case, 1000 events) a
 imperative that the container logs be free of these kinds of errors in production._ You can use the alternate HEC debug destination (below)
 to help debug this condition by sending direct "curl" commands to the HEC endpoint outside of the SC4S setting.
 
+## Invalid SC4S listen ports
+
+[SC4S can exclusively grant some port to device by using environment var `SC4S_LISTEN_{vendor}_{product}_{TCP/UDP/TLS}_PORT={port}`](https://splunk.github.io/splunk-connect-for-syslog/main/sources/#unique-listening-ports).
+
+During startup SC4S validating that listen ports configured correctly and in case of misconfiguration will see it on logs.
```

Comment:
During startup, SC4S validates that listening ports are configured correctly, and in case of misconfiguration, you will be able see any issues in (which log?) logs.